apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-notice.tsx

```diff
@@ -105,37 +91,18 @@ export const LivePreviewUpgradeNotice: FC< {
 					: [] ),
 			],
 		} );
-		return () => removeNotice( UPGRADE_NOTICE_ID );
```

Comment:
This is removed because we will refresh the page anyway after upgrading the plan. Correct?

Reply:
Yes!

Reply:
More specifically, the `* Show the notice when the canvas mode is 'view'.` part is not idempotent, so it needs the condition to prevent inconsistency. 
So I removed cleanup functions rather than coordinating them.